docs/cryoet_data_portal_docsite_FAQ.md

```diff
@@ -1,4 +1,209 @@
+
+## How do I use the Portal's API to select data?
+
+Every class in the Data Portal API has a `find` method which can be used to select all objects that match criteria provided in a query. The `find` method utilizes python comparison operators `==`, `!=`, `>`, `>=`, `<`, `<=` as well as `like`, `ilike`, and `_in` methods used to search for strings that match a given pattern, to create queries. 
+
+- `like` is a partial match, with the % character being a wildcard
+- `ilike` is similar to like but case-insensitive
+- `_in` accepts a list of values that are acceptable matches.
+
+The general structure of these commands is `class.find(client, query_filters=[LIST QUERIES HERE])`. For example, the script below will print the names of all runs that have "ts" in their name and more than 900 pixels in their "fast" axis.
+
+```python
+from cryoet_data_portal import Client, Run
+
+# Instantiate a client, using the data portal GraphQL API by default
+client = Client()
+
+# Query the Run class for runs with "TS" (case-insensitive) in their name and x pixels > 900
+runs_list = Run.find(client, query_filters=[Run.name.ilike("%TS%"), Run.tomogram_voxel_spacings.tomograms.size_x > 900])
+
+for run in runs_list:
+    print(run.name)
+```
+
+For more examples of using the `find` operator, check out the [tutorial here](https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_quick_start.html#python-quick-start). The Data Portal API reference can be found [here](https://chanzuckerberg.github.io/cryoet-data-portal/python-api.html).
+
+## What is the dataset identifier and portal ID?
+
+The dataset identifier in the API refers to the Portal ID provided in the Portal. This number is assigned by the Data Portal as a unique identifier for a dataset and is used as the directory name in the data filetree.
+
+Descriptions of all terminology and metadata used in the Portal is provided [here](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).
+
+## How do I contribute data to the Portal?
+
+Thank you for considering submitting data to the Portal! 
+
+Contributions can be raw data (tilt series and movie frames) + resulting tomograms, a new tomogram for existing raw data in the Portal generated using a different algorithm, and/or annotations of existing tomograms. We will work with you directly to upload the data to the Portal. Please fill out [this contribution survey](ADD LINK TO SURVEY), which is also found through the `Tell Us More` button on the bottom of Portal pages. We will then reach out to you to start the process of uploading your data. We have a ~6 month release cycle, so please allow time for the data to become available through the portal. 
```

Comment:
Consider adding that contributing lower quality tilt series, etc. is also useful/encouraged to support methods development

Reply:
I agree with @kandarpksk, this would be a useful addition. Lack of lower quality/more difficult data is something that methods developers have complained about at CZII workshops, so we should encourage this.

Reply:
How is this phrasing:
"
We encourage all contributions, including those which may be of lower quality than existing datasets on the Portal, as these datasets are useful for developing better annotation and data processing algorithms. 
"